### PR TITLE
Documentation for export/import a task

### PR DESCRIPTION
**Pull request by TOsmanov**
### Motivation and context

This PR contains:
- Documentation for export/import a task, also images for this section

This PR must be accepted after #3364

### Checklist

- [x] I submit my changes into the `develop` branch
- [ ] I have added description of my changes into [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [x] I have updated the [documentation](
  https://github.com/opencv/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [x] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2021 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

**Review by azhavoro — DISMISSED** [review]

**Review by kirill-sizov — DISMISSED** [review]

File: site/content/en/docs/manual/advanced/export-import.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+linkTitle: 'Export/import'
+weight: 25
+---
+
+In CVAT you can export and import tasks.
+This can be used to backup the task on your PC or to transfer the task to another server.
+
+## Export task
+
+To export the task, open the action menu and select `Export Task`.
```
Comment (bsekachev) [inline site/content/en/docs/manual/advanced/export-import.md:12]:
```suggestion
To export a task, open the action menu and select `Export Task`.
```

File: site/content/en/docs/manual/advanced/export-import.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+│   ├── manifest.jsonl
+├── task.json
+└── annotations.json
+```
+
+Export task API:
+
+- endpoint: `/api/v1/tasks/{id}?action=export​`
+- method: `GET`
+- returns: zip archive
```
Comment (bsekachev) [inline site/content/en/docs/manual/advanced/export-import.md:31]: @azhavoro 

Probably it returns 201, then 200, and finally 202 codes. Isn't it?
I mean that result is not always zip archive. Should we specify it here?

File: site/content/en/docs/manual/advanced/export-import.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+## Import task
+
+To import the task go to the tasks page, click the `Import Task` button and select the file you need.
```
Comment (bsekachev) [inline site/content/en/docs/manual/advanced/export-import.md:35]:
```suggestion
To import a task from an archive, go to the tasks page, click the `Import Task` button and select the archive you need.
```

File: site/content/en/docs/manual/advanced/export-import.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+![](/images/image220.jpg)
+
+As a result, you'll get a task containing data, parameters, and annotations of a previously exported task.
```
Comment (bsekachev) [inline site/content/en/docs/manual/advanced/export-import.md:39]:
```suggestion
As a result, you'll get a task containing data, parameters, and annotations of the previously exported task.
```

File: site/content/en/docs/manual/advanced/export-import.md [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+  ```
+
+## Import project (not implemented)
+
+Import project API:
+
+- endpoint: `/api/v1/projects?action=import​`
+- method: `POST`
+- Content-Type: `multipart/form-data​`
+- returns: json
```
Comment (bsekachev) [inline site/content/en/docs/manual/advanced/export-import.md:74]: Should we write in user guide about not implemented features?

Comment (azhavoro) [reply site/content/en/docs/manual/advanced/export-import.md]: This part has been moved from hld and I hope it will be implemented soon :)

**Review by bsekachev — APPROVED** [review]

**Pull request merged**